ui/widgets/tooltip.js

```diff
@@ -331,7 +331,7 @@ $.widget( "ui.tooltip", {
 					position( positionOption.of );
 					clearInterval( delayedShow );
 				}
-			}, $.fx.interval );
+			}, 13 );
```

Comment:
Do we want a comment here why its 13?

Reply:
Good point. This is copied from the `$.fx.interval` value & follows the advice from https://github.com/jquery/jquery-migrate/blob/master/warnings.md#jqmigrate-jqueryfxinterval-is-deprecated